gpm_core/src/package.rs

```diff
@@ -57,6 +68,79 @@ impl PackageInformation {
             extra_data: Vec::new(),
         }
     }
+
+    /// return all the information required for published package
+    /// ([`RequiredPublishInformation`]), otherwise, return an error containing a
+    /// list of the missing field
+    pub fn required_publish_information(&self) -> Option<RequiredPublishInformation> {
+        let creator = if let Some(creator) = &self.creator {
+            creator.to_string()
+        } else {
+            return None;
+        };
```

Comment:
```suggestion
        let creator = self.creator.cloned()?;
```
This should do the trick as well. Same treatment for the other invocations below.

Reply:
That's perfect (with ``.clone()``)